src/common/Popup.js

```diff
@@ -5,9 +5,10 @@ import { View, Dimensions, StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
   popup: {
-    marginLeft: 20,
-    marginRight: 20,
-    margin: 2,
+    position: 'absolute',
+    left: 20,
+    right: 20,
```

Comment:
I was trying to keep `Popup` more generic (not restricting its position, so it might be used be somewhere else too. Currently not sure where :P)

> That way, it is less effort to fix, less effort to review, and more likely to not introduce new issues.

completely agreed 😄 

Thanks! learning a lot while working :)